### PR TITLE
feat(standalone): expose olm package as parameters

### PR DESCRIPTION
**Pull request by machi1990**
## Description

Follows up a discussion with @daveconde on setting up standalone clusters

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has created for changes required on the client side

### CODE REVIEW

**Comment by machi1990** [comment]
hey @daveconde mind giving this a look?

File: Makefile [hunk 1]
```diff
@@ -723,6 +723,8 @@ deploy/service: OBSERVABILITY_CONFIG_TAG ?= "main"
 deploy/service: DATAPLANE_CLUSTER_SCALING_TYPE ?= "manual"
 deploy/service: STRIMZI_OPERATOR_ADDON_ID ?= "managed-kafka-qe"
 deploy/service: KAS_FLEETSHARD_ADDON_ID ?= "kas-fleetshard-operator-qe"
+deploy/service: STRIMZI_OLM_PACKAGE_NAME ?= "managed-kafka"
+deploy/service: KAS_FLEETSHARD_OLM_PACKAGE_NAME ?= "kas-fleetshard-operator"
```
Comment (JameelB) [inline Makefile:727]: Can you add a description for these 2 fields in the docs as well please :) https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/docs/deploying-kas-fleet-manager-to-openshift.md#deploy-kas-fleet-manager

Comment (machi1990) [reply Makefile:727]: Make sense. I'll mark them as optional.

**Review by JameelB — APPROVED** [review]
Thanks for this @machi1990! :)

**Pull request merged**